modules/ROOT/pages/multi-tenancy-and-tenant-configuration.adoc

```diff
@@ -3,14 +3,23 @@
 
 [WARNING]
 ====
+This feature was deprecated in Bonita v2021.1. This means that Multi-tenant architecture will continue to work in Bonita for existing Bonita subscription customers,
+but it will not be offered to new customers, nor offered as a modification for existing Bonita subscription customers who are not already using multi-tenancy.
+====
+
+[NOTE]
+====
+Important note: As mentioned in the release note, Bonita Enterprise customers need a contractual agreement to create additional tenants. For any new tenant creation, any general or contractual question related to Multi-tenancy, please contact your customer success representative.
```

Comment:
about 'important note', remove or put bold. Currently this is hidden

![image](https://user-images.githubusercontent.com/27200110/140988138-72b3f3b8-733b-48dc-be7e-488ae89a0091.png)

Reply:
The IMPORTANT admonition is here for that...